server/Controllers/editprofile.js

```diff
@@ -0,0 +1,18 @@
+import UserModel from "../Schemas/User";
+
+const editUserInfo = async (req, res) => {
+  // todo
+  // const id = req.userId;
+
+  try {
+    const userInfo = await UserModel.findByIdAndUpdate(req.userId, req.body, {
```

Comment:
1 - There's no email verification ! 😱 
----------
You can have `toto` as an email
You don't check if the email is already taken
And you don't send any email to check if the user own the new email address.

You need to the same as in the signup. Something like that:
1. use `validEmail`
It's in `Controllers/signUp.js`, yeah in Controllers, I never understood why... (jennifer) So put it in `Helpers/signUp.js` it make more sense ! and use it
2. use `emailIsFree`
It's also in `Controllers/signUp.js`, so same thing, put it in `Helpers/signUp.js` and use it
3. Update the user: BUT we don't want to loose the old email. Save the new email in a new property (like `newEmail`), not `email`
3. use `sendValidateEmail`.
This time it's already in `Helpers/signUp.js`. It take the users and the email language as parameter.
You'll need to change a bit the function, we want to send an email to `newEmail` not `email` (something like `to: user.newEmail || user.email` should work). For the language, you need to send the language the user was using on the website from the Front. 
(3.2. Bonus: it would be cool to send an email to the users email to inform them that they are changing their email)
4. Later, email verification (when the user clink on the link we sent him)
It happen in `verifyEmail` in `Controllers/signUp.js`. You'll need to change it a bit: if `emailVerified` is already `true`, you should set `email` as `newEmail` and remove `newEmail`.

oh, and you need to send the front error messages if the email is taken. And warn the user that he must confirm his email (maybe with a `Snackbar` ?)

**Then**, you're good for the email :)

-----------

2 - You can change all the property 😱
------------
With Postman, I was able to change my `password` 🙃 (no hashing, my password is now "caca" 👏🏻)
You can also change `picture` and `emailVerified` (and that should not be possible)